binary-compatibility-validator/test/PublicApiTest.kt

```diff
@@ -68,3 +72,26 @@ class PublicApiTest(
             error("No single file matching $regex in $libsDir:\n${files.joinToString("\n")}")
     }
 }
+
+private val ATOMIC_FU_REF = "Lkotlinx/atomicfu/".toByteArray()
+
+private fun JarFile.checkForAtomicFu() {
+    var found = false
+    for (e in entries()) {
+        if (e.name.endsWith(".class")) {
```

Comment:
nit: reverting this condition will make the code a bit more readable (one less indentation level)

Reply:
Good catch. I've found a way to save one more indent level, too: `val bytes = getInputStream(e).use { it.readBytes() }`